### PR TITLE
Enable Running BatchSupport Sample with NuGet

### PR DESCRIPTION
**Pull request by nums11**
This change enables  running the BatchSupport sample with NuGet to avoid the current issues of the BatchSupport sample with Inbox WinML breaking on prior versions of Windows.

### CODE REVIEW

File: Samples/BatchSupport/BatchSupport.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.1000
```
Comment (tiagoshibata) [inline Samples/BatchSupport/BatchSupport.sln:4]: I samples are at VS 2017

File: Samples/SampleSharedLib/SampleSharedLib/SampleSharedLib.vcxproj.filters [hunk 2]
```diff
@@ -29,8 +29,8 @@
     <ClCompile Include="FileHelper.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="dllload.cpp">
-      <Filter>Source Files</Filter>
-    </ClCompile>
```
Comment (tiagoshibata) [inline Samples/SampleSharedLib/SampleSharedLib/SampleSharedLib.vcxproj.filters:34]: Was it removed on purpose?

File: Samples/BatchSupport/BatchSupport.sln (hunk 1 above)
Comment (nums11) [reply Samples/BatchSupport/BatchSupport.sln:4]: Do I just need to update my version of Visual studio? I thought I was on the latest for Visual Studio 2019

File: Samples/SampleSharedLib/SampleSharedLib/SampleSharedLib.vcxproj.filters (hunk 2 above)
Comment (ryanlai2) [reply Samples/SampleSharedLib/SampleSharedLib/SampleSharedLib.vcxproj.filters:34]: Yes, in fact I would even recommend that the dllload.cpp file should just be removed. This was originally used to be able to load our inbox Windows.AI.Machinelearning.dll from local folder as the app. However, now we have NuGet as a solution, we don't want to have this backdoor way of loading our inbox binaries in our sample.

File: Samples/BatchSupport/BatchSupport.sln (hunk 1 above)
Comment (ryanlai2) [reply Samples/BatchSupport/BatchSupport.sln:4]: I'm fine with making this sample Visual Studio 2019 but then we'll need to update the README documentation of this sample to indicate that 2019 is needed.

Comment (nums11) [reply Samples/BatchSupport/BatchSupport.sln:4]: I'm fine with keeping it at VS 2017 to be consistent with the rest of the samples but I'm not sure how to edit this file. From what I understand this is solution file is generated and used by Visual Studio so it's not regularly editable. I figure I could edit in NotePad or a regular text editor but is that the best solution?

Comment (ryanlai2) [reply Samples/BatchSupport/BatchSupport.sln:4]: Yeah just edit solution file.

Please keep the original : # Visual Studio 15
VisualStudioVersion = 15.0.28307.1000

**Review by tiagoshibata — APPROVED** [review]

**Pull request merged**